.github/workflows/lint.yml

```diff
@@ -0,0 +1,26 @@
+name: Linting
+
+on: [push]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ["3.9", "3.12"]
```

Comment:
Not sure it matters but minimum version is 3.10 now.

Does the lint check depend on the Python version?

Reply:
ok. I think the cython-lint does not, but maybe `ruff` checks will depend if they are added later. So let us keep a set of python versions

Reply:
now the lint workflow is running ; probably it will fail, let's see